erpnext/stock/doctype/item_price/item_price.py

```diff
@@ -55,6 +55,11 @@ def validate_item(self):
 		if not frappe.db.exists("Item", self.item_code):
 			frappe.throw(_("Item {0} not found.").format(self.item_code))
 
+		if self.uom and not frappe.db.exists(
+			"UOM Conversion Detail", {"parenttype": "Item", "parent": self.item_code, "uom": self.uom}
```

Comment:
Should it still throw if it's the same as the default uom?

Reply:
Last time I checked, the default UOM was automatically added to the UOM conversion table. If that's no longer the case I'll need to fix this.

Reply:
Seems to still be the case

Reply:
It could still be raised if the row is deleted from the item.

It's throwing in our test cases:
https://github.com/resilient-tech/india-compliance/actions/runs/8432302882/job/23091138778?pr=1923#step:12:124

We shall figure out the exact issue and fix if required.

Reply:
It's probably not created from the backend.

https://github.com/resilient-tech/india-compliance/blob/fedca09c11a040781163c6d766b01c0d34382df7/india_compliance/gst_india/overrides/test_transaction.py#L206

Reply:
This line in your test case probably prevents the relevant code being triggered:

https://github.com/resilient-tech/india-compliance/blob/fedca09c11a040781163c6d766b01c0d34382df7/india_compliance/gst_india/overrides/test_transaction.py#L215